scripts/test.py

```diff
@@ -128,25 +128,25 @@ def parse_filter(pattern_src):
 
 
 def filter_tests(tests,
-                 whitelist_filters,
-                 blacklist_filters,
+                 includes_filters,
+                 excludes_filters,
                  permutations=None):
     if permutations is None:
         permutations = tests
 
-    whitelist_patterns = list(map(parse_filter, whitelist_filters))
-    blacklist_patterns = list(map(parse_filter, blacklist_filters))
+    includes_patterns = list(map(parse_filter, includes_filters))
+    excludes_patterns = list(map(parse_filter, excludes_filters))
 
     def keep(profile, test):
-        whitelisted = all(
+        includesed = all(
```

Comment:
name is a bit funny ... is that the result of a global replace?

Reply:
lol yes it is.